### ISSUE TITLE
create README profile page for IOOS github organization

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/.github#1]
create `/profile/README.md` which documents who we are, what we do, and what purpose the ioos github organization serves.

cc @mwengren @ocefpaf

**Comment by MathewBiddle** [comment]
see https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile

**Comment by MathewBiddle** [comment]
We should add info about whom to email to add new collaborators too.

**Comment by ocefpaf** [comment]
Here is conda-forge's .github repo in case we want to steal some ideas: https://github.com/conda-forge/.github

**Comment by mwengren** [comment]
I created a new repository form (I think), in this PR: https://github.com/ioos/.github/pull/2.  

We can link to this from the README page somehow to point people to fill out this form when requesting a new repository be created.  

Unfortunately, I couldn't see how to get the GH form preview to work when creating the yaml file, so I don't know what it looks like or if it works as expected.  I think it needs to be merged first before it can be previewed.

**Comment by MathewBiddle** [comment]
if you enable issues in your fork, you should be able to see what it looks like.

**Comment by MathewBiddle** [comment]
![image](https://user-images.githubusercontent.com/8480023/191760672-a0c12451-375e-47a7-a1e3-72bd91b87f43.png)

**Comment by MathewBiddle** [comment]
Something we didn't expect:

I went to create an issue in another repository under the IOOS Org (https://github.com/ioos/ioos-atn-data/issues/new/choose) and it looks like this template comes up there too. Is that a feature or a bug?

![image](https://user-images.githubusercontent.com/8480023/191767861-92d72575-a62c-4ee3-bd31-ad554cd299a2.png)

**Comment by mwengren** [comment]
It looks like the templates/forms placed in https://github.com/ioos/.github/tree/main/.github/ISSUE_TEMPLATE are applied across the organization.  It wasn't quite clear how that would work.  Maybe we need to rethink where to put the form I created since we don't really want it to appear in every repo most likely? 

Maybe we need a dedicated repo for business tasks like this?  I agree with the [suggestion](https://github.com/ioos/.github/pull/3#discussion_r977662254) to create another form for adding new contributors as well, rather than listing our email addresses. But we should figure out where best to put these first.

Here's the link to the [issue forms documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) and the [form schema](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema) I followed when creating these.  Note they're listed in beta at the moment.

**Comment by ocefpaf** [comment]
> It looks like the templates/forms placed in https://github.com/ioos/.github/tree/main/.github/ISSUE_TEMPLATE are applied across the organization.

Yep. That are org wide. One can override them with another template in the repo in question.

> Maybe we need a dedicated repo for business tasks like this? 

Maybe a GH discussion topic that is always open?

**Comment by mwengren** [comment]
I'm leaning towards a dedicated 'business' or some other appropriately named repository where we can host issue templates for specific requests/actions like this (and delete the org-wide form/template I just created here).  

Assuming we're expecting more than just this one request type, of course, and also that there isn't another better hack-y way we haven't thought of yet.  

Continuously open discussions could work but I like how the forms can request/require specific input (like .gitignore file type, etc).  

I did a test of single-repo issue templates and it worked (or at least it could override the org-wide one I created).  

See: https://github.com/ioos/sensorml2iso/issues/new?assignees=&labels=repo&template=new-repository.yml (formatting is slightly different).  The template lives here: https://github.com/ioos/sensorml2iso/blob/master/.github/ISSUE_TEMPLATE/new-repository.yml.

**Comment by MathewBiddle** [comment]
@mwengren I agree - a dedicated business repository to manage these requests seems the most appropriate. We can still have the IOOS org README point to using that ticket template in the business repo.

**Comment by mwengren** [comment]
@MathewBiddle How about a repo called 'governance'?  Or something like that?

Some of the details that I added to the IOOS Org README in [this PR](https://github.com/ioos/.github/pull/3/files) should move there instead (basically everything after the 'How To' header), and the IOOS Org README will just link to the 'governance' repo.

It's kind of a hack to create a repository only to handle GH Org-related issues/requests like this, but I don't see another way to get this information in a standardized way.

I'd say if there are other issue templates/forms we can see adding to the 'governance' repo in the future, it's worth doing.  Not sure it would be if it's only for this one repo creation request, though.  What do you think?

**Comment by MathewBiddle** [comment]
Yeah, I think if we are going to do it, that's a good approach. One comment on the name, maybe `gh-governance` or `github-governance` to make it more clear that the repo will only be addressing governance of GitHub stuff, not other IOOS 'things'?

ATM it would contain two issue templates:
1. Repository creation
2. Adding users to IOOS Org teams

But, I could see it as a place where we could include Community of Practice guidelines, references to other governance topics (licensing comes to mind). So, I guess I could see a couple other things being added there. Perusing the GitHub repos, I found this example of a governance repository https://github.com/concourse/governance - I guess there are a lot of things we could do with it?

As an aside - I think the IOOS Org README could contain a curated ToC of tools/packages/resources in the IOOS GH org. Linking to compliance checker and the web tool, ioos code lab, etc. to make it a little easier for folks to find the things we want them to find.

@ocefpaf do you have experiences with other orgs and a `governance` repository, outside of what you've mentioned already?

**Comment by mwengren** [comment]
Thanks for the input.  I think I prefer the simplicity of just `governance` for the name, following that example you found.  

I'd like to bite the bullet and create it now so we can make some progress before the end of the week, we can change the name later if necessary.  

This might all be overkill since we don't get that many of these requests to begin with, but I'd like to think this is time well spent and will pay off down the road ultimately.

**Comment by MathewBiddle** [comment]
Works for me! 👍

**Comment by mwengren** [comment]
Done! https://github.com/ioos/governance.   I'll move the new-repository.yml issue form over there now and try to make a few more changes to my PR's.

**Comment by mwengren** [comment]
To Do:

- [ ] Create issue form in [ioos/governance](https://github.com/ioos/governance/tree/main/.github/ISSUE_TEMPLATE) to request to add new collaborators to IOOS Org teams
- [ ] Describe the two change request types and link to corresponding issue forms in ioos/goverance [README.md](https://github.com/ioos/governance/blob/main/README.md)
- [ ] Change IOOS profile [README.md](https://github.com/ioos/.github/blob/main/profile/README.md) to link to [ioos/governance](https://github.com/ioos/governance) and remove the change request details list there currently
- [ ] IOOS Org README could contain a curated ToC of tools/packages/resources in the IOOS GH org. Linking to compliance checker and the web tool, ioos code lab, etc. to make it a little easier for folks to find the things we want them to find

Completing the above should have a basic framework in place and we can close this issue out then.

**Comment by MathewBiddle** [comment]
@ocefpaf we should add something about pypi here too!

**Comment by ocefpaf** [comment]
> @ocefpaf do you have experiences with other orgs and a `governance` repository, outside of what you've mentioned already?

Yes and no. There are some ideas that make sense to port to IOOS, others are more specific to the orgs. I'll collect them into a doc and open an issue in the new repo @mwengren created.
